objects/rct2/music/rct2.music.fairground.json

```
@@ -0,0 +1,90 @@
+{
+    "id": "rct2.music.fairground",
+    "authors": ["Chris Sawyer", "Allister Brimble"],
+    "version": "1.0",
+    "originalId": "0000008E|#MUSIC01|00000000",
+    "sourceGame": ["rct2"],
+    "objectType": "music",
+    "properties": {
+        "rideTypes": ["merry_go_round"],
+        "tracks": [
+            {
+                "source": "$RCT2:DATA/css4.dat",
+                "name": "Children of the regiment",
```

This field should list the actual title, optionally with the title that RCT2 gives it in brackets.

```suggestion
                "name": "Die Regimentskinder (Children of the regiment)",
```